pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -608,6 +607,18 @@ func (npm *nominatedPodMap) AddNominatedPod(pod *v1.Pod, nodeName string) {
 	npm.Unlock()
 }
 
+// NominatedPodsForNode returns pods that are nominated to run on the given node,
```

Comment:
shouldn't the other two functions be removed now? ```PriorityQueue.NominatedPodsForNode``` and ```nominatedPodMap.podsForNode```

Reply:
Done.